website/src/components/TaskPage/TaskPage.tsx

```diff
@@ -15,26 +16,30 @@ type TaskPageProps = {
 export const TaskPage = ({ type }: TaskPageProps) => {
   const { t } = useTranslation(["tasks", "common"]);
   const taskApiHook = taskApiHooks[type];
-  const { tasks, isLoading, reset, trigger, error } = taskApiHook(type);
+  const { response, isLoading, completeTask, skipTask } = taskApiHook(type);
   const taskInfo = TaskInfos.find((taskType) => taskType.type === type);
 
-  if (isLoading) {
-    return <LoadingScreen text={t("common:loading")} />;
+  switch (response.status) {
+    case "AWAITING_INITIAL":
+      return <LoadingScreen text={t("common:loading")} />;
+    case "NONE_AVAILABLE":
+      return <TaskEmptyState />;
+    case "AVAILABLE":
+      return (
+        <>
+          <Head>
```

Comment:
Do you think we should move `<Head>` out of the switch statement, and only change the inner page content?